DataManagementSystem/Agent/FTSAgent.py

```diff
@@ -925,7 +925,10 @@ def __finalizeFTSJob( self, request, ftsJob ):
     if not processFiles["OK"]:
       log.error( processFiles["Message"] )
       return processFiles
-    ftsFilesDict = self.updateFTSFileDict( ftsFilesDict, processFiles["Value"] )
+    processFiles = processFiles['Value']
+    if processFiles['toRegister']:
+      log.error( "%f files could not be registered in FC" % len( processFiles['toRegister'] ) )
```

Comment:
I remind your again that log error messages must have a static part and a variable part as the first and the second argument. So, this line should rather be:
log.error( "A number of files could not be registered in FC", len( processFiles['toRegister'] ) )